content/building-blocks/analyze-data/non-parametric-tests/binomial.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Understanding the binomial test"
+description: "Short guide to understanding the workings of the binomial test and how to use it in R"
+keywords: ""
+weight: 1
+date:
+draft: false
+aliases:
+  - /binomial-test
+---
+
+
+## Overview
+Binomial test is very handy when dealing with a **binary data** with a **nominal** measurement scale. It answers the question: "Is the observed outcome significantly different from what we would expect by a random chance?"; such as: Testing if a coin is fair, testing if a an election outcome favours the Republican or the Democrats, etc.
```

Comment:
the front end could be a bit shorter -- focus mostly on the code and not the theory